chemprop/cli/utils/parsing.py

```diff
@@ -74,10 +78,15 @@ def get_column_names(
     rxn_cols: Sequence[str] | None,
     target_cols: Sequence[str] | None,
     ignore_cols: Sequence[str] | None,
+    splits_col: str | None,
+    weight_col: str | None,
     no_header_row: bool = False,
 ):
     df = pd.read_csv(path, header=None if no_header_row else "infer", index_col=False)
 
+    if no_header_row:
+        return ["SMILES"] + ["pred_" + str(i) for i in range((len(df.columns) - 1))]
```

Comment:
Can multicomponent be used with no_header_row set to True?

Reply:
No